src/App.jsx

```diff
@@ -1,12 +1,12 @@
 import './App.css'
-import ToDoContainer from './components/ToDoContainer/ToDoContainer'
+import TodoList from './components/ToDoContainer/ToDoContainer'
 
 function App() {
 
   return (
     <>
       <main className='h-screen m-auto flex justify-center items-center bg-gradient-to-r from-green-300 via-teal-400 to-blue-500'>
-        <ToDoContainer />
+        <TodoList />
       </main>
     </>
```

Comment:
just a renaming suggestions, instead of Container, from the App perspective this is , in fact, the TodoList , 

you can even add an `index.js` file in your `components/` folder exporting **just** the Todolist component, keeping "private" the rest of modules/components/files